### DISCUSSION TITLE
Githib Education - Teacher Developer Pack benefit disappeared

### DISCUSSION THREAD

**Discussion by zkostic** [community/community#84673] [GitHub Education]
### Hi everyone,

I am a University Professor - doing research and teaching large classes using github.

I have been (and am) a verified member of the Github Education Teacher program.
For many years, that allowed me to get the free tools as a teacher benefit. Specifically, I was using GitKraken Client tool.
In 2024 - that seems to have disappeared.
When I go to https://education.github.com/teachers, the benefits do not include developer tools.

Is is intentional that teachers lost access to developer tools?
If not, please provide links how to get them.
If yes, a good question to ask is: How can we teach tools if students have access and we do not?

**Comment by gary-quinsac** [comment]
Hello zkostic,

It seems that I faced the same issue as you did. At the beginning of 2024, when starting GitKraken Client tool, a message welcomed me by claiming that I was on a free licence and that I could start the 7-day trial for the pro version. However, I was connected using the same teacher-verified account that had given me access to the pro version for more than a year.

I solved my problem by reinstalling GitKraken Client tool. I hope this can work for you as well. In the meantime, you can check at this address if you still have access to GitKraken Client Pro: https://app.gitkraken.com/purchase-license.
 
Regards,

Gary

**Comment by wiep** [comment]
Facing the same issue, I clicked on my GitHub name in the top right corner of the client and selected Account -> Sign into different account -> Sign in with GitHub. After completing the authentication on GitHub, in the account settings [1] it states

> You have access to GitKraken Client Pro and GitLens Pro through the GitHub Teacher Pack

and GitKraken continues to work with the pro features.

[1] https://app.gitkraken.com/purchase-license

**Comment by simaremare** [comment]
I think teachers are [no longer eligible for the tools](https://docs.github.com/en/education/about-github-education/github-education-for-teachers/why-wasnt-my-application-to-github-education-for-teachers-approved).

**Comment by anbrav0** [comment]
Hello all

I have been under the Github Educators teacher license for a while now and I have noticed that I no longer have access to GitKraken Pro. Was this resources taken away for educators?

**Comment by JannemanDev** [comment]
same here
